resources/recipes.py

```diff
@@ -277,6 +278,10 @@ def plank(thing: str):
             if item_data.type == 'all' or item_data.type in metal_data.types:
                 heat_recipe(rm, ('metal', '%s_%s' % (metal, item)), 'tfc:metal/%s/%s' % (item, metal), metal_data.melt_temperature, None, '%d tfc:metal/%s' % (item_data.smelt_amount, melt_metal))
 
+    metal_data = METALS['wrought_iron']
+    heat_recipe(rm, 'raw_bloom', 'tfc:raw_iron_bloom', metal_data.melt_temperature, None, '100 tfc:metal/cast_iron')
+    heat_recipe(rm, 'refined_bloom', 'tfc:refined_iron_bloom', metal_data.melt_temperature, None, '100 tfc:metal/wrought_iron')
```

Comment:
```suggestion
    wrought_iron = METALS['wrought_iron']
    heat_recipe(rm, 'raw_bloom', 'tfc:raw_iron_bloom', wrought_iron.melt_temperature, None, '100 tfc:metal/cast_iron')
    heat_recipe(rm, 'refined_bloom', 'tfc:refined_iron_bloom', wrought_iron.melt_temperature, None, '100 tfc:metal/cast_iron')
```
Better variable name, and melting should never melt into wrought iron. "Molten Wrought Iron" should be (effectively) unobtainable.